docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -3530,6 +3530,34 @@ To allow remote connections, set to `0.0.0.0`
 
 The maximum number of WebSocket connections allowed for JSON-RPC. Once this limit is reached, incoming connections are rejected. The default is 80.
 
+### `rpc-ws-max-frame-size`
+
+=== "Syntax"
+
+    ```bash
+    --rpc-ws-max-frame-size=<INTEGER>
+    ```
+
+=== "Example"
+
+    ```bash
+    --rpc-ws-max-frame-size=1024 * 1024
```

Comment:
you can't use this syntax
the example could just be --rpc-ws-max-frame-size=1024

Reply:
@macfarla The default value in the [file](https://github.com/hyperledger/besu/blob/main/besu/src/main/java/org/hyperledger/besu/cli/DefaultCommandValues.java#L63) follows that syntax. Also [here](https://github.com/hyperledger/besu/blob/main/ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/websocket/WebSocketConfiguration.java#L35).  Please confirm?

Reply:
yeah but in code 1024*1024 evaluates to an integer. It doesn't work that way on the command line (try it!). To get the same behaviour
--rpc-ws-max-frame-size=1048576